### ISSUE TITLE
Google Cloud Platform

### ISSUE THREAD

**Issue by Okachobe** [ashermancinelli/ape#11]
Working on creating the database on Google's cloud service.

**Comment by ashermancinelli** [comment]
Great. We have cloud functions setup for pull requests and a kubernetes service for deployment also on GCP.

**Comment by ashermancinelli** [comment]
- Where can we access this? 
- Could you post the ***temporary*** username/pw for the database so we can access it? 
- Is the schema the same and do you have a bucket for the dump?

**Comment by Okachobe** [comment]
35.199.183.112 is the address we can access the database from once connectivity issues are solved.

Issue 1: Unable to connect from school network due to it being a private it network - going to attempt from home and then continue debugging.
Issue 2: Setting up users and passwords for access. - Either find a way for the specific user to make their own passwords or collect what passwords everyone wants and then implementing them myself

Using schema from previous developers. have a virtual bucked on GCP containing MySQL dump file (SQL text file)

**Comment by Okachobe** [comment]
Using MySQL Workbench to connect to GCP Database when it is ready.

**Comment by Okachobe** [comment]
Issue 1. Attempted from home with the same issue looking for alternative solutions.

**Comment by ashermancinelli** [comment]
Have you whitelisted your machine's IP addr?

**Comment by Okachobe** [comment]
Yeah I whitelisted it from home with no VPN. and it didn't work. I am going to check out the link that you posted and try for a bit longer

**Comment by ashermancinelli** [comment]
Yeah I think you have to assign a static external IP, and I'm thinking that will do the trick.

**Comment by chazhurd** [comment]
146.187.0.43 IP

**Comment by ashermancinelli** [comment]
GCP MySQL won't save our ip addresses, we'll have to configure access from any ip @Okachobe
